<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Candy Crush</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/Resource/candyCrushLogo.png</normaloff>:/image/Resource/candyCrushLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_test">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>42</width>
      <height>42</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <disabledoff>:/image/Resource/origin/lollipop_withoutStick_red.png</disabledoff>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_backgroundMask">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_Candy">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; color:#ffeb05;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; color:#55ff00;&quot;&gt;n&lt;/span&gt;&lt;span style=&quot; color:#0055ff;&quot;&gt;d&lt;/span&gt;&lt;span style=&quot; color:#aa00ff;&quot;&gt;y&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_Crush">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa00ff;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; color:#0055ff;&quot;&gt;r&lt;/span&gt;&lt;span style=&quot; color:#55ff00;&quot;&gt;u&lt;/span&gt;&lt;span style=&quot; color:#ffeb05;&quot;&gt;s&lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;h&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gameStart">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>160</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resource/button_gameStart_background.png);
font: 75 20pt &quot;Arial&quot;;
color: rgb(9, 35, 3);
</string>
    </property>
    <property name="text">
     <string>Game Start!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setting">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_skip">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/Resource/fastForward.png</normaloff>:/image/Resource/fastForward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
